libs/@wingcloud/framework/src/main.ts

```diff
@@ -89,13 +89,15 @@ This is a Wing app and must be run through the Wing CLI (npm install -f winglang
   const rootId = process.env.WING_ROOT_ID;
 
   const $PlatformManager = new platform.PlatformManager({ platformPaths });
+  globalThis.$ClassFactory = $PlatformManager.createClassFactory();
```

Comment:
Perhaps move this into the createApp() implementation so at least it's reused across all entry sites.

Reply:
Ok, I've tried refactoring this a bit